Tests/Common/SonarQube/TestLogger.cs

```diff
@@ -75,17 +71,23 @@ public void AssertErrorLogged(string expected)
             bool found = this.Errors.Any(s => expected.Equals(s, System.StringComparison.CurrentCulture));
             Assert.IsTrue(found, "Expected error was not found: '{0}'", expected);
         }
-
-        public void AssertMessageNotLogged(string message)
+        
+        /// <summary>
+        /// Checks that no message contain all of the specified strings
```

Comment:
Changed the check to handle multiple message fragments. The new tests don't actually call this method, but I changed it to be consistent with *AssertWarningNotLogged* below is used by the new tests.